### ISSUE TITLE
Namespace Polishing

### ISSUE THREAD

**Issue by maggieliu** [rancher/dashboard#1784]
List/Create => pending on project v2 scope

Details:

- [ ] 1.  State boxes display all the resources within that Namespace states
- [ ] 2. ~Events should be there.~ Conditions and Related Resources
- [ ] 3. Remove the resource limit tabs

**Comment by maggieliu** [comment]
wireframe linked [here](https://github.com/rancher/dashboard/issues/1752#issuecomment-724990080)

**Comment by sadiapoddar** [comment]
verified on 2.5-head, commit be9683f
A detail view has been added for the namespace view which shows
   - number of resources,
   - status
   But there is no events information on the detail page, I created a new deployment in this namespace there is no information about it in the message.

Reopening this bug to add the "events" for this namespace. 

The detail page of a namespace. 

<img width="1508" alt="Screen Shot 2021-01-15 at 9 15 56 PM" src="https://user-images.githubusercontent.com/72409237/104798037-2ac24980-5778-11eb-951f-75ba201a78e8.png">
